src/services/mongoService.ts

```diff
@@ -1,18 +1,30 @@
 import { logger } from "@/src/utils/logger";
-import * as dotenv from "dotenv";
+import { env } from "node:process";
 import mongoose from "mongoose";
 
-dotenv.config();
+const { MONGODB_USERNAME, MONGODB_PASSWORD, MONGODB_URL, MONGODB_DATABASE, MONGODB_PORT } = env;
 
-const url = process.env.MONGODB_URL;
-
-if (url === undefined) {
+if (MONGODB_USERNAME === undefined) {
+    throw new Error("MONGODB_USERNAME not found. Make sure you have a .env file in the root of the project!");
+}
+if (MONGODB_PASSWORD === undefined) {
+    throw new Error("MONGODB_PASSWORD not found. Make sure you have a .env file in the root of the project!");
+}
+if (MONGODB_URL === undefined) {
     throw new Error("MONGODB_URL not found. Make sure you have a .env file in the root of the project!");
 }
+if (MONGODB_DATABASE === undefined) {
+    throw new Error("MONGODB_DATABASE not found. Make sure you have a .env file in the root of the project!");
+}
+if (MONGODB_PORT === undefined) {
+    throw new Error("MONGOD_PORT not found. Make sure you have a .env file in the root of the project!");
+}
```

Comment:
should we use something like zod to check if its an string?

Reply:
I agree with you 100%. I just haven't had time to implement it. I've been busy with my IRL job.